crates/db_schema/src/utils.rs

```diff
@@ -567,6 +575,20 @@ impl<RF, LF> Queries<RF, LF> {
   }
 }
 
+pub fn visible_communities_only<Q>(my_person_id: Option<PersonId>, query: Q) -> Q
```

Comment:
We *could* add this as a trait and impl to `PersonId`, but it seems kind of overkill, especially since its the only one.